RELEASE.md

```diff
@@ -27,7 +33,6 @@ Please follow the established format:
 - Improve layer solving approach when layers partially defined. (#407)
 - Remove 'code' flag to enable the code panel feature by default. (#404)
 - Remove 'lazy' flag to enable lazy loading of the sidebar by default. (#404)
-- Render the pipeline with warning of parameters missing from the catalog instead of showing an obfuscated error. (#43)
```

Comment:
I wonder why is this deleted from the release notes? Ideally the past release notes should not be deleted but instead any new updates should be reflected in the later release notes